src/template.yml

```diff
@@ -1601,9 +1662,7 @@ Resources:
                   - organizations:ListAccounts
                   - organizations:ListAccountsForParent
                   - organizations:DescribeAccount
-                  - organizations:ListOrganizationalUnitsForParent
-                  - organizations:ListRoots
-                  - organizations:ListChildren
+                  - organizations:List*
```

Comment:
Instead of removing the specific actions required, can you add the action that was missing?